models/Header.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+// Header : Represent a json model Header
+type Header struct {
+	Orientation string    `json:"orientation,omitempty"` // Can be "col" or "row", default "col"
```

Comment:
It may be easier to have this information implicitly added a given struct name

For example:

```go
type Row Column

type Column struct {
    Header *Header ...
    Data []*Data ...
}
```